cmd/rpcdaemon/commands/bor_snapshot.go

```diff
@@ -193,6 +193,88 @@ func (api *BorImpl) GetCurrentValidators() ([]*valset.Validator, error) {
 	return snap.ValidatorSet.Validators, nil
 }
 
+type BlockSigners struct {
+	Signers []difficultiesKV
+	Diff    int
+	Author  common.Address
+}
+
+type difficultiesKV struct {
+	Signer     common.Address
+	Difficulty uint64
+}
+
+func (api *BorImpl) GetSnapshotProposerSequence(number *rpc.BlockNumber) (BlockSigners, error) {
```

Comment:
We should use this as input param just like in bor `blockNrOrHash *rpc.BlockNumberOrHash`

Reply:
Done